solution/build.sh

```diff
@@ -97,6 +97,72 @@ function copy_image()
         dir:${FULL_PATH}
 }
 
+function generate_local_dashboard()
+{
+    if [[ $# -ne 1 ]]
+    then
+        echo "missing argument, $0 <dashboard>"
+        exit 1
+    fi
+
+    file=$1
+    filename=$(basename ${file})
+
+    dashboard_base_dir=${IMAGES_ROOT}/${filename%.json}-dashboard/${VERSION}/
+    mkdir -p ${dashboard_base_dir}
+
+    # get sha256, sha256sum prints the checksum and the filename, keep the checksum only
+    digest=$(sha256sum  ${file} | cut -d " " -f 1)
+    # get only the size (in bytes) of the file. format "%s" only prints the size
+    size=$(stat --printf "%s" ${file})
+
+    # generate empty config file
+    config_file=${dashboard_base_dir}/config.json
```

Comment:
should not "blindly" a (temp) file in the source code, it may conflict with an existing file (we never know...) : use mktemp instead